pkg/scheduler/service/run.go

```diff
@@ -196,5 +206,13 @@ func (r *RunRemote) Run(ctx context.Context) error {
 		}
 	}()
 
+	//start cleaner
```

Comment:
I'm not sure that the name cleaner is good enough for a component like this :D Any suggestion?

Reply:
I like it! :)